Running cluster for deep fits
180105 180106 normal
Running object 180105
Not enough g observations!
WARNING: Could not memory map array with mode='readonly', falling back to mode='denywrite', which means that the array will be read-only [astropy.io.fits.file]
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  25
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  48
Min gap is: 0.01171875
Number of observations now:  26
Counts: {'r': 24, 'i': 46, 'z': 25}
Mu: {'g': 0.0, 'r': 0.108486384, 'i': 0.20583369, 'z': 0.35075507}
No handles with labels found to put in legend.
Variable??
0
0
Non-Variable source in g
Variable??
0.01623779513873516
1.1324139762964125
Non-Variable source in r
Variable??
0.028631617961917225
1.4662927566655015
Non-Variable source in i
Variable??
0.07491313852369785
1.557427169339609
Non-Variable source in z
Variable:    0/4
Var Bands:   
Slope:       [1, 1, 1, 1]
Slope Error: [0.0, 0, 0.3949795042489067, 0.2560631090067088]
intercepts:  [0, 0, -0.1687079242082094, -0.1701092630481469]
Int Error:   [0.0, 0, 0.047858183618024824, 0.06162429873155552]
StDev:       [0.0, 0.06634715, 0.055337247, 0.19865228]
Not variable, closing plots & going to next row
Successfully finished!
Rows 180105 to 180106 ran in 3.3061 seconds
